File: CS/XtraReportApp/Form1.cs
using DevExpress.DataAccess.Sql;
using System;

namespace XtraReportApp {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
            reportDesigner1.AddService(typeof(IDBCommandInterceptor), new NolockInterceptor());
        }
        private void Form1_Load(object sender, EventArgs e) {
            reportDesigner1.OpenReport(new XtraReport1());
        }
    }
}

File: VB/XtraReportApp/Form1.vb
Imports DevExpress.DataAccess.Sql
Imports System

Namespace XtraReportApp
	Partial Public Class Form1
		Inherits DevExpress.XtraEditors.XtraForm

		Public Sub New()
			InitializeComponent()
			reportDesigner1.AddService(GetType(IDBCommandInterceptor), New NolockInterceptor())
		End Sub
		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
			reportDesigner1.OpenReport(New XtraReport1())
		End Sub
	End Class
End Namespace
